lib/service/steps_inspector.rb

```diff
@@ -48,6 +48,7 @@ def step_result
     end
 
     def result_emoji
+      return "💥" if raised_exception?
```

Comment:
That way, we can differentiate between a “classic” failed step inside a `try` block or a step that raised an exception.

Reply:
This makes sense to me :+1: